Source/JavaScriptCore/interpreter/Interpreter.cpp

```diff
@@ -1125,9 +1124,9 @@ ALWAYS_INLINE JSValue Interpreter::executeCallImpl(VM& vm, JSObject* function, c
     return JSValue::decode(vmEntryToNative(nativeFunction.taggedPtr(), &vm, &protoCallFrame));
 }
 
-JSValue Interpreter::executeCall(JSGlobalObject* lexicalGlobalObject, JSObject* function, const CallData& callData, JSValue thisValue, const ArgList& args)
+JSValue Interpreter::executeCall(JSObject* function, const CallData& callData, JSValue thisValue, const ArgList& args)
 {
-    VM& vm = lexicalGlobalObject->vm();
+    VM& vm = VM::fromInterpreter(this);
```

Comment:
How about adding `vm()` method to Interpreter which returns VM& via this offset computation?

Reply:
I guess I can that is more ergonomic.  I'll add it.

Reply:
Turns out there's already a `Interpreter::vm()` that does the exact same thing but is in the `.cpp`.  I'll make it inlineable,